quickstart/201-k8s-cluster-with-tf-and-aks/ssh.tf

```diff
@@ -0,0 +1,25 @@
+resource "random_pet" "ssh_key_name" {
+  prefix    = "ssh"
+  separator = ""
+}
+
+resource "azapi_resource" "ssh_public_key" {
+  type      = "Microsoft.Compute/sshPublicKeys@2022-11-01"
+  name      = random_pet.ssh_key_name.id
+  location  = "westus3"
+  parent_id = azurerm_resource_group.rg.id
+}
+
+resource "azapi_resource_action" "ssh_public_key_gen" {
+  type        = "Microsoft.Compute/sshPublicKeys@2022-11-01"
+  resource_id = azapi_resource.ssh_public_key.id
+  action      = "generateKeyPair"
+  method      = "POST"
+
+  response_export_values = ["publicKey"]
+}
+
+output "key_data" {
```

Comment:
Could we move this `output` block to `outputs.tf` file?

Reply:
@lonegunmanb I'd like to keep the SSH.tf file fully encapsulated so that I can easily copy it to other projects. If I split some of its functionality across multiple files, that task becomes more difficult as I then have to remember to copy the file and update another file.